testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automated Release Process
+
+This feature is to automate the design system release process, removing repeatable manual developer actions. 
+
+As a design system developer 
+I need to be able to specify that a design system release is Alpha or Public
+So that users understand that the design system is ready for either the public or develop environments
+ 
+As a design system developer 
+I need to have the option to release design system updates manually
+So that I am not reliant on 3rd party services to release updates automatically e.g. Github / Jenkins
+ 
+As a design system developer 
+I need to be able to stop the automated design system release process
+So that so that I can prevent errors and make manual changes after the process has started
+ 
+As a user of the design system
+I need to be aware of new design system releases and any changes
```

Comment:
This is drawn from the 'ideal process' but may be a requirement we want to capture.

User could be defined more narrowly as: "as a member of the #design-system slack channel" if this is too vague currently

Reply:
I think `what has changed since the last release` instead of `any changes` would be more explicit.

Reply:
updated, thanks.